EngineHacks/SkillSystem/Skills/TurnSkills/Installer.event

```diff
@@ -14,7 +14,10 @@ WORD HoardersBaneID
 VulneraryItemID_Link:
 WORD Vulnerary 
 
-
+SandstreamID_Link:
```

Comment:
Please don't use this syntax for function label names. `SandstreamID_Link` implies that the byte following this label will be the value of `SandstreamID`, not the function. Convention would name this label `Sandstream`, or if that symbol is already in use `SandstreamFunc`.